src/console/sudokuio.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+class SudokuIO(object):
+
+    def __init__(self,file_path):
+        """ Constructor of the class, curently this define the variables for the open 
+        mode w and w+ and the path where the file is located.
+
+        string file_path - this is the current path where the path is located.
+        """
+        self.file_path = file_path
+        self.WRITE_MODE_TRUNCATE = 'w+'
+        self.WRITE_MODE = 'w'
+
+    def write(self, text):
+        """ Method that write a text in the file located in the path self.file_path
+
+        string text - this is the text that will be saved in the file.
+        """
+        file = open(self.file_path, self.WRITE_MODE_TRUNCATE)
+        file.write(text)
+        file.close()
+
+    def read_all(self):
+        """ Method that read all content of a text file and return the string value for
+        the content.
+
+        return string - return an string with the content of all file.
+        """
+        file = open(self.file_path)
+        result = file.read()
+        file.close()
+        return result
+
+    def delete_content(self):
+        """ Method that delete the content of a file or clear the content of the file 
+        located in the path self.file_path
+        """
+        open(self.file_path, self.WRITE_MODE).close()
+
+    def delete_file(self):
+        """ This method remove the file from the system        
+        """
+        if self.exist_the_file():
+            os.remove(self.file_path)
+
+    def exist_the_file(self):
```

Comment:
the isfile method does not verify if the file exists it verifies if it is a file or directory or something else, this implementation is not correct